cypress/integration/pages/article/canonical.js

```diff
@@ -176,6 +176,36 @@ Object.keys(config)
         });
       });
 
+      describe('ATI', () => {
+        it('should have a noscript tag with an 1px image with the ati url', () => {
+          cy.hasNoscriptImgAtiUrl(
+            envConfig.atiUrl,
+            config[service].isWorldService
+              ? envConfig.atiAnalyticsWSBucket
+              : '',
+          );
+        });
+      });
+
+      describe('Chartbeat', () => {
```

Comment:
Chartbeat is toggled off at the moment on live. Are we going to expect Cypress to fail when things are toggled off?

Reply:
Gah, Fixed